tools/ci_build/build.py

```diff
@@ -338,6 +338,8 @@ def parse_arguments():
         help="Specify the minimum version of the target platform "
         "(e.g. macOS or iOS)"
         "This is only supported on MacOS")
+    parser.add_argument("--disable_graph_partition", action='store_true',
```

Comment:
Let's put this under --use_openvino and change help to: 
"**Used with OpenVINO EP.** Only runs fully supported graphs on OpenVINO EP else the model fallbacks to default CPU EP."